prestodb/hdp2.6-hive/Dockerfile

```diff
@@ -36,7 +36,15 @@ RUN yum install -y \
 
 # Cleanup
   && yum -y clean all && rm -rf /tmp/* /var/tmp/* \
-  && ln -s /usr/share/java/mysql-connector-java.jar /usr/hdp/current/hive-metastore/lib/mysql-connector-java.jar
+  && ln -s /usr/share/java/mysql-connector-java.jar /usr/hdp/current/hive-metastore/lib/mysql-connector-java.jar \
+
+# Remove extra not needed packages
+  && rpm -e --nodeps $(rpm -qa | grep '^pig') \
```

Comment:
`--nodeps` sounds like "i wanna hurt myself"

is `$(rpm -qa | grep '^pig')` equivalent to `'pig*'`?

Reply:
> --nodeps sounds like "i wanna hurt myself"

Exactly ; / `hive` depends on `pig` somehow.